python/paddle/nn/layer/common.py

```diff
@@ -342,3 +343,59 @@ def forward(self, input):
             mode=self._mode,
             pad_value=self._pad_value,
             data_format=self._data_format)
+
+
+class AlphaDropout(layers.Layer):
+    """
+        :alias_main: paddle.nn.AlphaDropout
+        :alias: paddle.nn.AlphaDropout,paddle.nn.layer.AlphaDropout,paddle.nn.layer.common.AlphaDropout
+
+    AlphaDropout Layer.
+    This interface is used to construct a callable object of the `AlphaDropout` class.
+    Alpha Dropout is a type of Dropout that maintains the self-normalizing property. For an input with
+    zero mean and unit standard deviation, the output of Alpha Dropout maintains the original mean and
+    standard deviation of the input. Alpha Dropout fits well to SELU activate function by randomly setting
+    activations to the negative saturation value.
+
+    For more information, please refer to:
+    [Self-Normalizing Neural Networks](https://arxiv.org/abs/1706.02515)
+    `paddle.nn.functional.alpha_dropout`
+
+    In dygraph mode, please use ``eval()`` to indicate whether it is in test phrase or not.
```

Comment:
to switch to evaluation mode, where dropout is disabled.

Reply:
Done